datafusion/functions/src/datetime/to_char.rs

```diff
@@ -172,6 +172,19 @@ fn _to_char_scalar(
     let data_type = &expression.data_type();
     let is_scalar_expression = matches!(&expression, ColumnarValue::Scalar(_));
     let array = expression.into_array(1)?;
+
+    if format.is_none() {
+        if is_scalar_expression {
+            return Ok(ColumnarValue::Scalar(ScalarValue::Utf8(
+                Some(String::new()),
+            )));
+        } else {
+            return Ok(ColumnarValue::Array(Arc::new(StringArray::from(
+                vec![String::new(); array.len()],
```

Comment:
I haven't wrapped the values using ScalarValue::Utf8(Option<String>) because all the other instances were also directly using `Vec<String>` to create the `StringArray`. 

I'm not sure if it matters for an empty string. Let me know if there's a difference in this case though.